### ISSUE TITLE
Element becomes untweenable after tween scale to NaN

### ISSUE THREAD

**Issue by dondiegote** [greensock/GSAP#453]
Hello gsap team.

I found a problem with gsap 3.x. When you try tween an element scale to NaN, gsap silently fails and the element becomes untweenable.

Steps to reproduce:
```

var num = 0/0;
TweenMax.to(".rec", 1, {scale:num});

TweenMax.to(".rec", 1, {scale:.1});    // don´t work
TweenMax.to(".rec", 1, {x:200});    // don´t work

num = 2;
TweenMax.to(".rec", 1, {scale:num});    // don´t work, element is untweenable

```
Expected behaviour:
- Keep the element tweenable
or
- Throw Error, assuming it is an error trying to scale to NaN.

Thanks for this great library, have a nice day.

**Comment by jackdoyle** [comment]
Thanks for pointing that out, @dondiegote. I definitely wouldn't recommend setting thing to invalid values, but I can add some logic to the next release that senses that condition and works around it (defaulting to 0). You can preview the next release (minified) here: https://assets.codepen.io/16327/gsap-latest-beta.min.js

Better?

**Comment by dondiegote** [comment]
I agree Jack, definitely not a good idea tween nothing to NaN.
I used to apply some transforms with gsap.set (lazy me), and in my actual proyect one previous operation returned me NaN sometimes. I spent some time checking why it didn't apply tweens after that, so i'm pretty sure this little change can helps other users to save time.

I checked the last version, works perfectly.

Thx again, you guys do a great job with gsap.

Have a nice day
Diego
